set.mm

```diff
@@ -422469,7 +422470,7 @@ that confuses text editors (such as emacs).
        <li><b>Typography conventions.</b>
        Class symbols for functions (e.g., ` abs ` , ` sin ` )
        should usually not have leading or trailing blanks in their
-       HTML/Latex representation.
+       HTML representation.
```

Comment:
Why are these typography conventions not valid for Latex representations? If not one-to-one, then there should be corresponding conventions for Latex.

Reply:
Spaces have no effect in LaTeX's math mode (that is, `$a a$` and `$aa$` both give $aa$).

Reply:
> Spaces have no effect in LaTeX's math mode (that is, `$a a$` and `$aa$` both give aa).

OK, but if spaces are required to improve the readability in LaTeX, they must be inserted/enforced explicitly (e.g., by \quad, \, etc.). See, for example the LaTeX representation of the operator `gcd`: latexdef "gcd" as "\,{\rm gcd}\,";
By this, it is ensured that the operation `( a gcd b )` is not displayed as "(agcdb)" , but as "(a gcd b)". Therefore, there must be a convention for LaTex  corresponding to the convention for HTML.

Reply:
> OK, but if spaces are required to improve the readability in LaTeX, they must be inserted/enforced explicitly (e.g., by \quad, \, etc.).

If that's for readibility of the generated latex code, then it's the job of the code generator to separate tokens with spaces, not for the definition of each token to contain spaces.

Enforcing spaces explicitly like you propose is bad practice (spaces don't scale well, for one).

For instance, `\,{\rm gcd}\,` is an example of what not to do. Since gcd is used in set.mm as a binary operation, it should be `\mathbin{\operatorname{gcd}}`.